Built classification models & sentiment analysis

jesseg102 opened the pull request:
Here are various classification models w/ performance metrics

jesseg102 committed:
Built classification models & sentiment analysis

jesseg102 committed:
Add files via upload

Separated classification models into individual chunks

shailja-somani-0 reviewed (approved):
Overall looks great! Some discrepancies with the vectorization &  classification model metrics that we discussed in Slack, but those were fixed in the latest commit on my branch, so let's just merge this in & then I'll merge mine in!

jesseg102 committed:
Classification Model Improvement

I was able to fix the categorization predictive model based on newly introduced data. There were some outputs that differed from what was shared, but the adjustments ran on my computer
